crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -207,6 +207,18 @@ impl<'w, T: Component> Deref for Res<'w, T> {
     }
 }
 
+impl<'w, T: Component> AsRef<T> for Res<'w, T> {
+    fn as_ref(&self) -> &T {
+        &*self
+    }
+}
+
+impl<'w, T: Component> AsMut<T> for ResMut<'w, T> {
+    fn as_mut(&mut self) -> &mut T {
+        &mut *self
+    }
+}
+
```

Comment:
could you put this impl next to `impl<'w, T: Component> AsRef<T> for ResMut<'w, T>` below?

Reply:
Good call, not sure how that got mixed up